packages/react/src/components/WhatsNew/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carbon-labs/react-whats-new",
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
You can keep this at `0.0.0` (or `0.0.1`) since it’ll get incremented during the actual release.